packages/blockchain-wallet-v4-frontend/src/modals/Brokerage/Banks/Withdraw/EnterAmount/template.success.tsx

```diff
@@ -90,7 +138,7 @@ const Success: React.FC<InjectedFormProps<
 
   const userCanWithdraw =
     Number(props.withdrawableBalance) > Number(props.fees.value)
-  const showFee = Number(props.fees.value) > 0
+  const showFee = Number(props.fees.value) > 0 && transferAccount === undefined
```

Comment:
i could see someone setting `transferAccount` to null or something in the future. maybe `&& !transferAccount` would be more versatile here?